.changes/v2.20.0/550-improvements.md

```diff
@@ -0,0 +1 @@
+* Added support for using subnet prefix length while creating vApp networks, fixed accessing uninitialized Features field while updating a vApp network. [GH-550]
```

Comment:
Should this one be split in two log entries?

Reply:
Fixed.